client/packages/inventory/src/Stocktake/DetailView/ChangeLocationModal.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import {
+  InputWithLabelRow,
+  useTranslation,
+  ConfirmationWithChildrenModal,
+} from '@openmsupply-client/common';
+import {
+  LocationRowFragment,
+  LocationSearchInput,
+} from '@openmsupply-client/system';
+import { useStocktake } from '../api';
+
+interface ChangeLocationConfirmationModalProps {
+  isOpen: boolean;
+  onCancel: () => void;
+}
+
+export const ChangeLocationConfirmationModal = ({
+  isOpen,
+  onCancel,
+}: ChangeLocationConfirmationModalProps) => {
+  const t = useTranslation('inventory');
+
+  const [location, setLocation] = useState<LocationRowFragment | null>(null);
```

Comment:
Would it be worth adding logic so if either only 1 line selected, or if all lines selected have same location, to show that location here instead of blank? Should it show the like `[multiple]` thing otherwise?

Reply:
I like the blank tbh